feature/utils.py

```diff
@@ -78,6 +78,36 @@ def normalize_columns(df: pd.DataFrame):
     return df_normalized.fillna(0)
 
 
+def _process_data(input_df: pd.DataFrame, categories: List[str], feature_column: str, selection_size: int) \
```

Comment:
This isn't being used anymore? I'm not following the relationship between this function and the _process_data() in text_base.py. It looks like they are doing different things though.

Reply:
commenting this out for now to avoid confusion (and taking back into text_based.py file)